core/domain/suggestion_registry.py

```diff
@@ -1223,6 +1224,51 @@ def set_translation_suggestion_count_for_language_code(
         """
         self.translation_suggestion_counts_by_lang_code[language_code] = count
 
+    def are_translation_reviewers_needed_in_lang_code(self, lang_code):
+        """Returns whether or not more reviewers are needed to review
+        translation suggestions in the given language code.
+
+        Args:
+            lang_code: str. The language code of the translation
+                suggestions.
+
+        Returns:
+            bool. Whether or not more reviewers are needed to review
+            translation suggestions in the given language code.
+       """
+        if lang_code not in self.translation_suggestion_counts_by_lang_code:
+            return False
+
+        if lang_code not in self.translation_reviewer_counts_by_lang_code:
+            return True
+
+        number_of_reviewers = (
+            self.translation_reviewer_counts_by_lang_code[lang_code])
+        number_of_suggestions = (
+            self.translation_suggestion_counts_by_lang_code[lang_code])
+        return python_utils.divide(
+            number_of_suggestions, number_of_reviewers) > (
```

Comment:
It would be cleaner to multiply instead. That way you don't have to deal with integer/fractional and rounding issues.

Reply:
Done